python/srv

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2017 CodiLime
+# Copyright 2016-2017 CodiLime
```

Comment:
There are two reasonable options:
1. Set the dates based on creation `date` - `last change of file`
2. Set the dates - everywhere the same 2016 - `last commit to repository`

As this repo history started in 2017, then either we drop 2016, or change all 2017 with 2016.

Manual updates are not an option IMO